rust/tool/src/cli.rs

```diff
@@ -30,6 +31,10 @@ enum Commands {
 
 #[derive(Parser)]
 struct InstallCommand {
+    /// Target system
+    #[arg(long)]
+    target_system: String,
```

Comment:
```suggestion
    system: String,
```

Same as with `architecture`. Any cross compilation is handled beforehand within Nix/NixOS.

Reply:
I don't like this; unprefixed systems are the reason for why we are always confused. I am okay with dropping target, but I would like to have an alternative: host or build.

Reply:
That's fair but then we should use the same terms as Nixpkgs, maybe just `hostPlatform`? But `hotsPlatform` doesn't have a string representation, right?

Reply:
hostPlatform has a string representation